### PR TITLE
✏️ Remove strange example types

### PR DESCRIPTION
**Pull request by asim-shrestha**

### CODE REVIEW

File: server/server/__init__.py [hunk 1]
```diff
@@ -35,7 +33,6 @@ def fetch_all_examples() -> ExampleMeta:
     examples = get_all_examples()
     return ExampleMeta(
         count=len(examples),
-        goal_types=list(set([example.type for example in examples])),
```
Comment (asim-shrestha) [inline server/server/__init__.py:38]: this was never used

File: static/examples.json [hunk 2]
```diff
@@ -2817,141 +2817,6 @@
...
-    "evals": [
-      {
-        "type": "end_url_match",
-        "expected": "https://www.hackensackmeridianhealth.org/en/location-search#first=10&sort=%40locationname%20ascending&f:@locationtype=[Hospitals%20and%20Emergency%20Departments]"
-      }
-    ]
-  },
-  {
-    "id": "CMGxGp3IXTZDdl09hdA3X",
-    "url": "https://www.adventisthealthcare.com/locations/results/?searchId=7402b4ad-5a84-ee11-a85e-000d3a611ea2&sort=13&page=5",
```
Comment (asim-shrestha) [inline static/examples.json:2837]: We never used these tests

File: tests/test_example_eval.py [hunk 3]
```diff
@@ -175,7 +175,7 @@ def create_default_example(
 
 
 def test_non_fetch_with_goal() -> None:
-    example_data = create_default_example({"type": "multiple", "goal": "goal"})
+    example_data = create_default_example({"type": "links", "goal": "goal"})
```
Comment (asim-shrestha) [inline tests/test_example_eval.py:178]: `multiple` not a valid type anymore

**Comment by KhoomeiK** [comment]
lgtm

**Pull request merged**